### PR TITLE
improve look n feel of test output

### PR DESCRIPTION
**Pull request by stylewarning**

### CODE REVIEW

File: tests/compilation-tests.lisp [hunk 1]
```diff
@@ -168,7 +168,7 @@
 
 (deftest test-cnot-rewiring-in-cnot-architecture ()
   "Test that all CNOTs on each pair of qubits compile into a single CNOT rewired appropriately."
-  (format t "[Test output: ~%")
+  (format t "~&[    Test output: ~%")
```
Comment (appleby) [inline tests/compilation-tests.lisp]: This line is space-after-open-bracket, whereas below it's space-before-open-bracket. Intentional?

File: tests/compilation-tests.lisp [hunk 2]
```diff
@@ -186,7 +186,7 @@
 
 (deftest test-cz-compilation-and-rewiring-in-cnot-architecture ()
   "Test that all CZs on all qubit combinations compile to a single CNOT that's native."
-  (format t "[Test output: ~%")
+  (format t "~&    [Test output: ~%")
```
Comment (appleby) [inline tests/compilation-tests.lisp]: here

File: tests/compilation-tests.lisp (hunk 1 above)
Comment (notmgsk) [inline tests/compilation-tests.lisp]: This is different to the diff below. This has `[    Test...`, other has `     [Test....`.

Comment (notmgsk) [reply tests/compilation-tests.lisp]: Got there before ya ;D

Comment (appleby) [reply tests/compilation-tests.lisp]: Blast! Too slow.

**Pull request merged**